Update pull_request_template.md (#2)
HotStartup/.github: pull_request_template.md

<!-- 1. Main changes: New features, modules, fixes, or refactoring -->
<!-- 2. UI：見た目の変更点について、変更前後のスクリーンショットや動画を添付します -->

## 事前条件（Prerequisites）
<!-- レビュアーが動作確認をスムーズに行えるよう、前もって必要な情報があれば記載します -->

> <!-- In order to facilitate smooth testing by reviewers, list any necessary information in advance -->

Reviewer: DeepL 先生に直してもらいました
```suggestion
<!-- To facilitate a smooth review process, list all necessary information in advance -->
```

<!-- 例 / Example:
1. FeatureToggle
2. 他リポジトリの開発中ブランチ / Development branch of another repository
3. 特殊な再現手順など / Special reproduction steps, etc.
-->

## 影響範囲（Impact）
### 対象ユーザー（Affected Users）
<!-- 変更によって影響を与えるユーザーを記載します -->

> <!-- List the users affected by this change -->

Reviewer: ```suggestion
<!-- List the users who will be affected by this change -->
```

### 接点（Touchpoints）
<!-- 変更によって影響を与えるユーザーとの接点（画面、メールなど）を記載します -->

> <!-- Specify the touchpoints with users (e.g., screens, emails) that are affected by the change -->

Reviewer: ```suggestion
<!-- List the user touchpoints (e.g., screens, emails) that will be affected by this change -->
```
Affected Users と文言を揃えてみたり

### 機能（Features）
<!-- 変更によって影響を与えるユーザーへの提供機能を記載します -->

> <!-- Describe the features provided to users that are impacted by this change -->

Reviewer: ```suggestion
<!-- Describe the features that will be affected by this change -->
```
やや冗長な文章という雰囲気を感じ。日本語からそうかもしれない。
